src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -445,7 +472,11 @@ export class AddScenarioModal implements OnInit, OnDestroy {
         // add parameter name, description, add values/check if between limits
         regs.expected.parameters = {};
         for (const parameter of regRegs.parameters) {
+            if (this.skipCheck == true) {
+                regs.expected.parameters[parameter.code] = 0;
+            } else {
```

Comment:
This is fine, but if you're skipping the equation check it doesn't really matter if you have values or not, so you don't actually have to go through and remove these :)

Reply:
If I didn't have a value for the parameters it would tell me the equation was not valid. I tried in Insomnia to not have a value for the parameters and it wouldn't go through. I included the ways I tried and if they worked or not.
[works](https://user-images.githubusercontent.com/59063181/99092610-fb843600-2596-11eb-8025-dc2689ba2224.png)
[doesn't work](https://user-images.githubusercontent.com/59063181/99092751-2a9aa780-2597-11eb-82c3-3fbe4d4436dd.png)
[doesn't work](https://user-images.githubusercontent.com/59063181/99093004-8c5b1180-2597-11eb-9fff-ff6a8398c4b8.png)

Reply:
OK, gotcha!